src/components/Nav.js

```diff
@@ -0,0 +1,82 @@
+import "../styles/nav.css";
+import { useUrl } from "../context/useUrl";
+import { useCart } from "../context/cartContext";
+import { Link } from "react-router-dom";
+export default function Nav() {
+  const { wishList, cartItems } = useCart();
+
+  const { setUrl } = useUrl();
+
+  return (
+    <nav className="navbars">
+      <label className="label" htmlFor="toogle">
+        {" "}
+        &#9776;
+      </label>{" "}
+      <input type="checkbox" id="toogle" />
+      <label className="label-hide" htmlFor="toogle">
+        <div className="side-bar">
+          <ul>
+            <li
+              onClick={() =>
+                setUrl("https://api.rahulgupta99.repl.co/products")
+              }
+            >
+              All
+            </li>
+            <li
+              onClick={() =>
+                setUrl(
+                  "https://api.rahulgupta99.repl.co/products/category/women clothing"
+                )
+              }
+            >
+              Women's
+            </li>
+            <li
+              onClick={() =>
+                setUrl(
+                  "https://api.rahulgupta99.repl.co/products/category/men clothing"
+                )
+              }
+            >
+              Men's
+            </li>
+            <li
+              onClick={() =>
+                setUrl(
+                  "https://api.rahulgupta99.repl.co/products/category/electronics"
+                )
+              }
+            >
+              Electronics
+            </li>
+            <li
+              onClick={() =>
+                setUrl(
+                  "https://api.rahulgupta99.repl.co/products/category/jewelery"
+                )
+              }
+            >
+              Jewelery
+            </li>
+          </ul>
+        </div>
+      </label>
+      <ul>
+        <div style={{ position: "relative", marginLeft: "5px" }}>
```

Comment:
Avoid inline styles . They have high specificity. Will cause trouble in style changes